apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/shareEmbedModal/email-tab.tsx

```
@@ -0,0 +1,157 @@
+            </div>
+            <div>
+              <div className="mb-2 border-b border-slate-200 pb-2 text-sm">
+                {t("environments.surveys.share.send_email.email_to_label")} : {email || "user@mail.com"}
+              </div>
+              <div className="border-b border-slate-200 pb-2 text-sm">
+                {t("environments.surveys.share.send_email.email_subject_label")} :{" "}
+                {t("environments.surveys.share.send_email.formbricks_email_survey_preview")}
+              </div>
+              <div className="p-2">
+                {emailHtml ? (
+                  <div dangerouslySetInnerHTML={{ __html: emailHtmlPreview }}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)